complete/src/main/java/dto/TourGuide.java

```diff
@@ -0,0 +1,13 @@
+package TourGuide.dto;
+
+public class TourGuide {
+
+    private String KeyLocations;
+
+    public TourGuide(String keyLocations){
+        this.KeyLocations = keyLocations;
+    }
+
+    public String getKeyLocations() { return KeyLocations; }
+
+}
```

Comment:
added DTO and implemented Tour guide class that had a key location that is returned to the user. 
Source: https://martinfowler.com/articles/microservices.html

Reply:
Is this used?  Avoid adding unimplemented features, as unused code is clutter, which is technical debt.